README.md

```diff
@@ -1,11 +1,19 @@
 # Bazel rules for Jasmine
 
+Runs the [Jasmine](https://jasmine.github.io/) JS testing tool under Bazel.
```

Comment:
Do we want the "rules_* is just a part of what Aspect provides: ..." that we have in others such as jest?